### ISSUE TITLE
Caption controls not visible in high contrast mode desert theme

### ISSUE THREAD

**Issue by pehammer** [microsoft/devhome#1319]
### Dev Home version

0.301.198.0

### Windows build number

10.0.22621.1992

### Other software

OS Build Version: 10.0.22621.1992.amd64fre.ni_release.220506-1250
.NET Version: .NET 6.0.18

### Steps to reproduce the bug

Turn on high contrast, desert theme

<img width="1660" alt="image" src="https://github.com/microsoft/devhome/assets/112589934/5f90b768-77ca-4918-bf96-1d6cf58c5025">

### Expected result

caption controls

### Actual result

No caption controls

### Included System Information

Physical Memory: 63.72GB (30.95GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHome_0.301.198.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension_0.300.198.0_x64__8wekyb3d8bbwe

**Comment by krschau** [comment]
@pehammer Interesting, I'm not seeing this issue. Is there anything else about your settings that you think might be relevant?

![image](https://github.com/microsoft/devhome/assets/47155823/c47d1980-e710-4a11-8288-6d0081acaa25)

**Comment by krschau** [comment]
@pehammer Are you still seeing this bug?
/needinfo

**Comment by krschau** [comment]
Talked with @pehammer offline, he's still seeing it. Close buttons do show up on hover. App title is missing.

**Comment by krschau** [comment]
Duplicate of #1754
